app/content/models/user.py

```diff
@@ -104,7 +106,11 @@ def has_module_perms(self, app_label):
     def get_number_of_strikes(self):
         from django.db.models import Sum
 
-        aggregate_sum = self.strikes.all().aggregate(Sum("strike_size"))
+        aggregate_sum = self.strikes.filter(
+            created_at__lte=today() + timedelta(days=1),
+            created_at__gte=today() - timedelta(days=20),
```

Comment:
Noe grunn til ikke `datetime.now()`?
```suggestion
            created_at__lte=datetime.now(),
            created_at__gte=datetime.now() - timedelta(days=20),
```

Reply:
Samme ting bare today legger på tidzone oslo europe

Reply:
Tror du må ha `today()` siden serverklokken ligger 2 timer bak oss?

Reply:
Ahh skjønner